stable/drone/values.yaml

```diff
@@ -51,7 +51,10 @@ service:
 ingress:
   ## If true, Drone Ingress will be created.
   ##
-  enabled: false
```

Comment:
should the default change?

Reply:
Makes sense to me - will make the change

Reply:
Reverted to default value